src/main/java/com/gb/cornucopia/cookery/mill/TileEntityMill.java

```diff
@@ -45,6 +45,15 @@ public int getSizeInventory() {
 		return 9;
 	}
 
+	public boolean isEmpty() {
```

Comment:
isn't this a bit confusing also being called isEmpty? I am not even sure what it is used for yet though 😆